### PR TITLE
Egl atw

### PR DESCRIPTION
**Pull request by JeroMiya**
Experimental ATW implementation for Android OpenGL ES. Not recommended for regular use.

Issues preventing regular usage:
 - eglClientWaitSyncKHR extension potentially has driver issues on some Android builds.
 - VSync timing code needs improvement across all of the RenderManager backends.
 - Android Reprojection/Distortion-correction rendering code is very slow right now.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -182,7 +182,7 @@ endif()
 #-----------------------------------------------------------------------------
 # OpenGL library as a stand-alone renderer not wrapping D3D
 if ( ( (OPENGL_FOUND AND GLEW_FOUND) OR OPENGLES2_FOUND ) AND (SDL2_FOUND OR ANDROID))
-	list(APPEND RenderManager_SOURCES osvr/RenderKit/RenderManagerOpenGL.cpp osvr/RenderKit/RenderManagerOpenGL.h osvr/RenderKit/RenderManagerOpenGLC.cpp)
```
Comment (russell-taylor) [inline CMakeLists.txt]: The check for insertion of the OpenGLATW appears not to be strict enough.  The code fails to compile on Windows due to the following:
f:\taylorr\personal\work\consulting\sensics\osvr\src\osvr-rendermanager\osvr\renderkit\RenderManagerOpenGLATW.h(36): fatal error C1083: Cannot open include file: 'EGL/egl.h': No such file or directory

**Comment by JeroMiya** [comment]
@russell-taylor thanks for the code review! I added guards to cmake and source and tested the build on windows (with OpenGL enabled) and it now builds. Thanks for catching this.

**Comment by russell-taylor** [comment]
With these changes, it compiles and runs on a desktop nVidia with the non-directmode window mode.

**Comment by JeroMiya** [comment]
Thanks @russell-taylor!

**Pull request merged**